examples/meanio/README.md

```diff
@@ -0,0 +1,265 @@
+## MEAN.io example
+
+This example shows how to build a simple, multi-tier web application using Kubernetes and Docker.
+it is based on the [guestbook](https://github.com/GoogleCloudPlatform/kubernetes/tree/master/examples/guestbook) example.
+
+The example consists of:
+- A web framework using MEAN framework.
+- A MongoDB server.
+
+### Step Zero: Prerequisites
```

Comment:
The guestbook guide has since been updated, and this step (which appears to be the same) has changed. To avoid having to sync with that guide, maybe it'd be best to put a link here to that section of the guestbook guide?